contracts/src/protocol/OmniAVS.sol

```diff
@@ -220,7 +220,9 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, IServiceManager, OwnableUpgradeable
 
             uint96 total = _getTotalDelegations(operator);
             uint96 staked = _getSelfDelegations(operator);
-            uint96 delegated = total - staked;
+
+            // this should never happen, but just in case
+            uint96 delegated = total > staked ? total - staked : 0;
```

Comment:
@ttarsi thoughts? this happened in a tests cause I was manipulating state in eigenpod manager harness, but not correctly

Reply:
Yes, validate all data not bounded by our own system